numpy/random/_generator.pyx

```diff
@@ -717,6 +717,14 @@ cdef class Generator:
         out : bytes
             String of length `length`.
 
+        Notes
+        -----
+        This function generates random integers from a discrete uniform
+        distribution in the range [0, 2**32), converts them to little-endian
+        byte arrays, and then decodes the resulting byte array using the 
+        UTF-32 codec. The generated integers are interpreted as 32-bit 
```

Comment:
There are a few factual details here that are wrong, but before correcting them, I'm not really sure that this is something that needs to be documented.

Reply:
As this is a public function I would say it deserves some comment on how it works. There should at least be a comment on the uniformity of the distribution.

Reply:
I would confine the documentation to the properties of the results (e.g. uniformity, independence from the CPU's native endianness), not the step-by-step details of the implementation. We generally do not document such implementation details.